apps/duellen/index.html

```diff
@@ -3,98 +3,21 @@
   <head>
     <meta charset="utf-8">
     <title>Duellen</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons"rel="stylesheet">
-    <style>
-      @import url('https://fonts.googleapis.com/css?family=Open+Sans:400,600,700');
-
-      body{
-        width:100%;
-        position:relative;
-        font-family: 'Open Sans', sans-serif;
-      }
-
-      #wrapper{
-        width: 800px;
-        margin: auto;
-        background-color: white;
-        padding: 30px;
-      }
-
-      button{
-          text-decoration: none !important;
-          background-color: #f07e26 !important;
-          color: white !important;
-      }
-
-      #logout{
-
-        border: 0px;
-        cursor: pointer;
-        border-radius: 4px;
-        padding: 5px 20px;
-        margin-bottom: 10px;
-    
-      }
-
-      h1{
-        font-weight: 700;
-        font-size: 4em;
-      }
-
-      h2{
-        font-weight: 600;
-        font-size: 2.5em;
-      }
-
-      p{
-        font-weight: 400;
-        font-size: 2em;
-      }
-
-      p.header{
-        font-weight: 400;
-        text-transform: uppercase;
-        letter-spacing: 0.15em;
-      }
-
-      p.progress{
-        font-weight: 600;
-        text-transform: uppercase;
-        letter-spacing: 0.1em;
-        font-size: 0.9em;
-      }
-
-      p.facit{
-        font-weight: 600;
-        font-size: 1.1em;
-        letter-spacing: 0.1em;
-      }
-
-      .animation {
-          opacity: 1;
-          -webkit-animation: fade 5s linear forwards;
-          animation: fade 5s linear forwards;
-      }
-
-        @media only screen and (max-width: 460px) {
-          #wrapper{
-            width: 320px;
-          }
-        @media only screen and (max-width: 370px){
-          #wrapper{
-            width: 270px;
-          }
-        }
-      }
-
-    </style>
+    <link rel="stylesheet" type="text/css" href="styles.css">
   </head>
   <body>
-    <div id="wrapper">
-        <div id="root">
-      
-        </div>
-    </div>
+    <main class="container mt-5">
+      <div id="root">
+    
+      </div>
+    </main>
     <script type="text/javascript" src="index.js"></script>
+    <!-- Optional JavaScript -->
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
```

Comment:
We should add these scripts via `npm` instead of importing them like this, otherwise offline development wouldn't work